bloom-housing/bloom
# 2335/fixes adding user that's already created

## Issue Overview

This PR addresses [#issue](https://github.com/bloom-housing/bloom/issues/2335)


## Description
This fixes an error where if an email was already tied to a public user, then trying to add them as a partner was causing an error

## How Can This Be Tested/Reviewed?
Go into public site and add a new user 
go into the partner site and attempt to add that exact same user as a partner (there is currently discussion going on around if admin users should show in the user table)
the user should now be added correctly 

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
